### ISSUE TITLE
readXRefTable: unexpected eof

### ISSUE THREAD

**Issue by Demitroi** [hhrutter/pdfcpu#85]
Hi

I'm using the 0.1.23 version in Windows 10 AMD64

```
$ pdfcpu version
pdfcpu version 0.1.23
```

Validating the next PDF, i got the readXRefTable: unexpected eof error.

[01733D30CB6D4C6B9962BB715CAFE263.pdf](https://drive.google.com/file/d/1AJjVrE7yatn960R9oy2OJHYaAJNNheYx/view?usp=sharing)

The verbose output is the next

```
pdfcpu validate -vv C:\Users\SERGIO\Downloads\FWE940228N67_20190306155700.0807588\01733D30CB6D4C6B9962BB715CAFE263.pdf
validating(mode=relaxed) C:\Users\SERGIO\Downloads\FWE940228N67_20190306155700.0807588\01733D30CB6D4C6B9962BB715CAFE263.pdf ...
 INFO: 2019/06/01 13:21:46 reading C:\Users\SERGIO\Downloads\FWE940228N67_20190306155700.0807588\01733D30CB6D4C6B9962BB715CAFE263.pdf..
 READ: 2019/06/01 13:21:46 Read: begin
 INFO: 2019/06/01 13:21:46 PDF Version 1.5 conforming reader
 READ: 2019/06/01 13:21:46 readXRefTable: begin
 READ: 2019/06/01 13:21:46 scanning for offsetLastXRefSection starting at 115048
 READ: 2019/06/01 13:21:46 Offset last xrefsection: 114168
 READ: 2019/06/01 13:21:46 buildXRefTableStartingAt: begin
 READ: 2019/06/01 13:21:46 headerVersion begin
 READ: 2019/06/01 13:21:46 headerVersion: end, found header version: 1.2
 READ: 2019/06/01 13:21:46 newPositionedReader: positioned to offset: 114168
 READ: 2019/06/01 13:21:46 line: <xref >
 READ: 2019/06/01 13:21:46 buildXRefTableStartingAt: found xref stream
 READ: 2019/06/01 13:21:46 newPositionedReader: positioned to offset: 114168
 READ: 2019/06/01 13:21:46 parseXRefStream: begin at offset 114168
 READ: 2019/06/01 13:21:46 buffer: endInd=-1 streamInd=-1
 READ: 2019/06/01 13:21:46 buffer: endInd=-1 streamInd=-1
Fatal: EOF
readXRefTable: unexpected eof
github.com/hhrutter/pdfcpu/pkg/pdfcpu.readXRefTable
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/pkg/pdfcpu/read.go:1110
github.com/hhrutter/pdfcpu/pkg/pdfcpu.Read
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/pkg/pdfcpu/read.go:78
github.com/hhrutter/pdfcpu/pkg/pdfcpu.ReadFile
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/pkg/pdfcpu/read.go:57
github.com/hhrutter/pdfcpu/pkg/api.ReadContextFromFile
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/pkg/api/api.go:127
github.com/hhrutter/pdfcpu/pkg/api.Validate
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/pkg/api/api.go:141
github.com/hhrutter/pdfcpu/pkg/api.Process
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/pkg/api/process.go:84
main.process
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/cmd/pdfcpu/main.go:219
main.main
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/cmd/pdfcpu/main.go:182
runtime.main
        C:/Go/src/runtime/proc.go:200
runtime.goexit
        C:/Go/src/runtime/asm_amd64.s:1337
Read: xRefTable failed
github.com/hhrutter/pdfcpu/pkg/pdfcpu.Read
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/pkg/pdfcpu/read.go:80
github.com/hhrutter/pdfcpu/pkg/pdfcpu.ReadFile
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/pkg/pdfcpu/read.go:57
github.com/hhrutter/pdfcpu/pkg/api.ReadContextFromFile
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/pkg/api/api.go:127
github.com/hhrutter/pdfcpu/pkg/api.Validate
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/pkg/api/api.go:141
github.com/hhrutter/pdfcpu/pkg/api.Process
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/pkg/api/process.go:84
main.process
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/cmd/pdfcpu/main.go:219
main.main
        C:/Users/SERGIO/go/src/github.com/hhrutter/pdfcpu/cmd/pdfcpu/main.go:182
runtime.main
        C:/Go/src/runtime/proc.go:200
runtime.goexit
        C:/Go/src/runtime/asm_amd64.s:1337
```

Any idea? Thanks :)

**Comment by hhrutter** [comment]
Hi there! 
Please check if #82 solves your problem.

**Comment by hhrutter** [comment]
It' smth different. I am on it.

**Comment by hhrutter** [comment]
This is fixed in the upcoming release!

**Comment by hhrutter** [comment]
This is fixed with commit [86b3445](https://github.com/hhrutter/pdfcpu/commit/86b344515ab4f12e80ea26f3c0e519e28047274f).

**Comment by Demitroi** [comment]
In the latest commit works, Thanks :)
